[Failing-test] UT TestPrintIPAddressList for  leap day

### Which jobs are failing?

https://testgrid.k8s.io/sig-release-master-blocking#ci-kubernetes-unit
and also v1.27-1.29

### Which tests are failing?

TestPrintIPAddressList

### Since when has it been failing?

After 03-01

### Testgrid link

https://testgrid.k8s.io/sig-release-master-blocking#ci-kubernetes-unit

### Reason for failure (if possible)

```
{Failed;  === RUN   TestPrintIPAddressList
    printers_test.go:6616: mismatch:   []v1.TableRow{
          	{Cells: {string("192.168.2.2"), string("myresource.mygroup/mynamespace/myname"), string("10y")}},
          	{
          		Cells: []any{
          			string("2001:db8::2"),
          			string("myresource2.mygroup2/mynamespace2/myname2"),
        - 			string("5y1d"),
        + 			string("5y2d"),
          		},
          		Conditions: nil,
          		Object:     {},
          	},
          }
--- FAIL: TestPrintIPAddressList (0.00s)
;}
```

### Anything else we need to know?

the leap day caused the failure

### Relevant SIG(s)

/sig cli network

--- a/pkg/printers/internalversion/printers_test.go
+++ b/pkg/printers/internalversion/printers_test.go
@@ -6572,7 +6572,7 @@ func TestPrintIPAddressList(t *testing.T) {
 			{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:              "192.168.2.2",
-					CreationTimestamp: metav1.Time{Time: time.Now().AddDate(-10, 0, 0)},
+					CreationTimestamp: metav1.Time{},
 				},
 				Spec: networking.IPAddressSpec{
 					ParentRef: &networking.ParentReference{
@@ -6585,7 +6585,7 @@ func TestPrintIPAddressList(t *testing.T) {
 			}, {
 				ObjectMeta: metav1.ObjectMeta{
 					Name:              "2001:db8::2",
-					CreationTimestamp: metav1.Time{Time: time.Now().AddDate(-5, 0, 0)},
+					CreationTimestamp: metav1.Time{},
 				},
 				Spec: networking.IPAddressSpec{
 					ParentRef: &networking.ParentReference{
@@ -6600,8 +6600,8 @@ func TestPrintIPAddressList(t *testing.T) {
 	}
 	// Columns: Name, ParentRef, Age
 	expected := []metav1.TableRow{
-		{Cells: []interface{}{"192.168.2.2", "myresource.mygroup/mynamespace/myname", "10y"}},
-		{Cells: []interface{}{"2001:db8::2", "myresource2.mygroup2/mynamespace2/myname2", "5y1d"}},
+		{Cells: []interface{}{"192.168.2.2", "myresource.mygroup/mynamespace/myname", "<unknown>"}},
+		{Cells: []interface{}{"2001:db8::2", "myresource2.mygroup2/mynamespace2/myname2", "<unknown>"}},
 	}
 
 	rows, err := printIPAddressList(&ipList, printers.GenerateOptions{})

